typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -507,7 +566,7 @@ export const polygon: ChainMetadata = {
     {
       name: 'PolygonScan',
       url: 'https://polygonscan.com',
-      apiUrl: 'https://api.polygonscan.com',
+      apiUrl: 'https://api.polygonscan.com/api',
```

Comment:
Will this break our verification tooling?

Reply:
No because there was a corresponding change in MultiProvider